src/types.jl

```diff
@@ -1,15 +1,21 @@
-# Abstract types.
-abstract type BoundaryCondition end
-abstract type HomogeneousBoundaryCondition <: BoundaryCondition end
-abstract type InhomogeneousBoundaryCondition <: BoundaryCondition end
+# Types for Boundary Conditions
+    # Abstracts
+    abstract type BoundaryCondition end
+    abstract type HomogeneousBoundaryCondition <: BoundaryCondition end
+    abstract type InhomogeneousBoundaryCondition <: BoundaryCondition end
+    # Concretes
+    struct Reflecting <: HomogeneousBoundaryCondition end
+    struct Mixed{T} <: HomogeneousBoundaryCondition where {T <: Real}
+        ξ::T
+    end
+    struct Absorbing{T1, T2} <: InhomogeneousBoundaryCondition where {T1 <: Real, T2 <: Real}
```

Comment:
Can remove this until we are ready for it.

Reply:
Done.